src/CommunityToolkit/Xamarin.CommunityToolkit/Animations/TadaAnimation.shared.cs

```diff
@@ -5,7 +5,7 @@ namespace Xamarin.CommunityToolkit.Animations
 {
     public class TadaAnimation : AnimationBase
```

Comment:
If happy with the approach in `TadaAnimationType` this entire class and folder can disappear